src/Func.h

```diff
@@ -122,14 +120,8 @@ class Func : public Obj
 
 	virtual detail::TraversalCode Traverse(detail::TraversalCallback* cb) const;
 
-	uint32_t GetUniqueFuncID() const { return unique_id; }
-	static const FuncPtr& GetFuncPtrByID(uint32_t id)
-		{
-		return id >= unique_ids.size() ? Func::nil : unique_ids[id];
-		}
-
 protected:
-	Func();
+	Func() { }
```

Comment:
Use `= default` instead of `{ }`.

Reply:
Thanks!  I've made those changes and taken a stab at rebase'ing, plz see how it looks!